src/main/scala/com/campudus/tableaux/arguments.scala

```diff
@@ -62,7 +65,9 @@ object ArgumentChecker {
   }
 
   def isDefined(options: Seq[Option[_]], name: String = ""): ArgumentCheck[Unit] = {
-    val empty = !options.exists({ o => o.isDefined })
+    val empty = !options.exists({ o =>
+      o.isDefined
+    })
```

Comment:
The added newlines are probably caused by the `{...}` around the function, I guess. Does it behave the same way, when omitting them? Or just have `val empty = !options.exists(_.isDefined)`...